_posts/2018-03-26-idr.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: The Image Data Resource is added to the list of Scientific Data recommended data repositories
+categories: community
+---
+
+Nature Research journal [*Scientific Data*](https://www.nature.com/sdata/), is
+a peer-reviewed, open-access journal for descriptions of scientifically
+valuable datasets, and research that advances the sharing and reuse of
+scientific data. These data descriptors provide a path for publishing datasets
+associated with scientific publications.
+
+*Scientific Data* requires that all datasets related to a data descriptor,
+including experimental, computational and curated data, should be submitted to
+an appropriate open, public community repository. Although *Scientific Data*
+mandates the release of the datasets that accompany their manuscripts, they do
+not themselves host data. Instead, they encourage submission of data to
+community-recognized data repositories where possible.
+
+The [Image Data Resource](https://idr.openmicroscopy.org/about/) (IDR) has
+just been listed, by *Scientific Data*, as the recommended data repository for
+bioimaging data, the first for this data type. Authors wishing to submit a
+data descriptor that includes imaging data will be asked to deposit the data
+in IDR, as part of the manuscript submission process. IDR accepts reference
+image datasets (as defined  at the
```

Comment:
double space

Reply:
Suggestion: maybe remove 'the'?  ```(as defined at IDR Submissions)``` or, with 'the' ```(as defined at the IDR Submissions page)```